### ISSUE TITLE
Expired events stop working when decode-graphite-line return nil or malformed event

### ISSUE THREAD

**Issue by sblask** [riemann/riemann#338]
As described in https://github.com/aphyr/riemann/pull/335 , I don't get any expired events once Riemann received a malformed graphite message in which case decode-graphite-line will return nil or {:ok :true :service "exception"}. When logging all incoming events, I don't see nil, but I do see {:ok :true :service "exception"}. As memory consumption is not rising, I don't think the actual expiration breaks, maybe just triggering the event? Otherwise, it might actually break the indexing. I don't know how I would have to go about verifying either of them and the code is hard to follow. I'd appreciate any hints or ideas.

**Comment by rekhajoshm** [comment]
@aphyr  - We also see these errors on riemann - WARN [2015-02-09 12:12:17,691] pool-1-thread-4 - riemann.transport.tcp - TCP handler caught
java.lang.RuntimeException: Graphite server parse error (too many fields): "sh: proc-status-metrics.rb: command not found"

**Comment by aphyr** [comment]
Looks like your script is sending malformed input--e.g. an error message--to the graphite server.

**Comment by rekhajoshm** [comment]
thanks @aphyr seems like i had to use mutator on sensu handler
